assets/app/view/game_card.rb

```diff
@@ -188,7 +188,7 @@ def render_invite_link
 
       invite_url = url(@gdata)
       flash = lambda do
-        `navigator.clipboard.writeText((window.location + invite_url).replace('//game', '/game'))`
+        `navigator.clipboard.writeText((window.location.host + invite_url).replace('//game', '/game'))`
```

Comment:
This needs to be window.location.origin, not host. The host property doesn't include the scheme, so now the links are just "18xx.games/game/123" and don't linkify properly.

Reply:
can you please fix @magnusrk-ch ? thanks for identifying